mcs/class/corlib/System.IO/LogcatTextWriter.cs

```diff
@@ -48,13 +48,43 @@ public override void WriteLine ()
 			var o = line.ToString ();
 			line.Clear ();
 
+			Log (o);
+			stdout.WriteLine (o);
+		}
+
+		void Log (string message)
```

Comment:
Should be static

Reply:
It calls `Log(byte*)` which can't be static.